### PR TITLE
PLAT-118480: Fixed ImageItem width/height when orientation is horizontal

### PR DESCRIPTION
**Pull request by daniel-stoian-lgp**
### Checklist
* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [x] Documentation was verified or is not changed
* [x] UI test was passed or is not needed
* [x] Screenshot test was verified or is not needed

* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Issues: 
[Agate] ImageItem: image looks too flat when orientation is horizontal
[Agate] ImageItem: caption is above the small image when `orientation=horizontal` and `captionPosition=overlay`

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Solution: 
- added width/height for image when orientation is horizontal
- change styling so the `captionPosition=overlay` has no effect when `orientation=horizontal`. There is no use case for this combination.

### Links
[//]: # (Related issues, references)
PLAT-118480

### Comments

Enact-DCO-1.0-Signed-off-by: Daniel Stoian daniel.stoian@lgepartner.com

### CODE REVIEW

**Review by adrian-cocoara-lgp — APPROVED** [review]
LGTM

**Review by 0x64 — APPROVED** [review]
:lgtm:

File: ImageItem/ImageItem.module.less [hunk 1]
```diff
@@ -7,42 +7,48 @@
...
+			}
 		}
 	}
 
 	.applySkins({
 		border: @agate-imageitem-border-width solid @agate-imageitem-border-color;
 
+		&.horizontal {
+			.image {
+				margin: @agate-imageitem-horizontal-image-margin;
```
Comment (MikyungKim) [inline ImageItem/ImageItem.module.less:25]: We need to consider RTL here :)

**Review by MikyungKim — APPROVED** [review]
:shipit:

File: ImageItem/ImageItem.module.less (hunk 1 above)
Comment (daniel-stoian-lgp) [reply ImageItem/ImageItem.module.less]: right :)

**Pull request merged**